independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json

```diff
@@ -0,0 +1,89 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Default title",
+        "description": "",
+        "version": "1.0"
+    },
+    "host": "[YOURFUNCTIONAPP].azurewebsites.net",
```

Comment:
@Stephan-Rijk to use a dynamic host please setup the apiProperties.json file properly. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/policy-templates/dynamichosturl/dynamichosturl).